CPUserService/.gitignore

```diff
@@ -0,0 +1,33 @@
+HELP.md
```

Comment:
No need to Push .gitignore file.
Only files need to push are src/* , pom.xml

Reply:
ok